ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.10__amd64-stable.yaml

```diff
@@ -341,7 +341,6 @@ tests:
       TAG_VERSION: '@4.10'
     test:
     - chain: openshift-e2e-test-qe
-    workflow: cucushift-installer-rehearse-gcp-ipi-proxy-ccm
```

Comment:
The workflow is dropped here? Should be replaced?

Reply:
checking this, should be replaced.